frontend/src/Pages/FeedPage.tsx

```diff
@@ -37,7 +39,9 @@ export default function FeedPage() {
     const perpage = 20;
     const page = parseInt(search.get('page') || '1');
 
-    const { posts, loading, pages, error, updatePost } = useFeed(site, feedType, page, perpage);
+    const [changeSorting, setChangeSorting] = useState(-1);
```

Comment:
Nit: seems that `-1` doesn't have any special connotation here, probably better to replace it with `undefined`.